_pages/en/strategy-build-vs-buy.md

```diff
@@ -119,26 +119,27 @@ Stakeholders are expected to still comply with existing policy instruments inclu
 ### Governance, Compliance, and Reporting
 
 1. IT Solutions are broken down into manageable parts (IT Products)
-2. Each IT Product must be assessed on its own as well as a part of overall the IT Solution.
-3. IT Products may be bought, built, or adopted (e.g. Open Source Software)
-4. IT Solutions are assessed against the IT Solutions Risk Assessment Framework (see [Risk Managing the Obtention of Software](#risk-managing-the-obtention-of-software)).
+2. IT Products may be bought, built, or adopted (e.g. Open Source Software)
```

Comment:
Je pense pas qu'on devrait mettre OSS comme exemple pour adopter.  On pourrait aussi adopter un SaaS.
We could also "buy" OSS through an integrator and support provider.

Maybe we need to define Adopt, Build and Buy.

Reply:
Hmm, mais SaaS c'est acheter, non? À moins qu'on parle d'un SaaS version gratuite?

Reply:
En fait, tu achètes des services, pas du OSS comme tel...

Reply:
OK, je comprends ce que tu veux dire, je vais modifier pour rendre les exemples dans la définition à la place! :+1: